examples/pytorch/language-modeling/test_clm.sh

```diff
@@ -0,0 +1,15 @@
+python -m torch.distributed.launch --nproc_per_node=1 /cb/home/vinayr/ws/global_rigl/mlresearch/transformers/examples/pytorch/language-modeling/run_clm.py \
```

Comment:
maybe add a comment here referencing the graphcore hugging face config we are trying to match